### ISSUE TITLE
width issue with Table showing different Characters of another language 

### ISSUE THREAD

**Issue by ayonious** [ayonious/console-table-printer#308]
Originally reported by @RylanZhou

I've found that if the text to be displayed contains special characters, e.g., chinese, which actually takes up **two** spaces for **one** character, it will cause alignment problems. Here's an example:

```
┌────────────────────────────────────────────────────────────┬─────────────┐
│ SUMMARY                                                    │         KEY │
├────────────────────────────────────────────────────────────┼─────────────┤
│ Some Descriptions and Summary 这里是中文这里是中文这里是中文                │ ISSUE-19150 │
│ Some Summary 这里是中文这里是中文                │ ISSUE-18072 │
│ Description 这里是中文                │ ISSUE-17787 │
└────────────────────────────────────────────────────────────┴─────────────┘
```

By doing some research, I figured out that this is because the original `str.length` does not output the correct unicode length of a special character.

```javascript
'char'.length; // Outputs 4, correct
'中'.length; // Outputs 1, but it actually contains 2 unicodes
```

The same problem also occurs when calling `str.padEnd()` and `str.padStart()`.

Hoping to fix this for better compatibility, I've installed a dependency called `wcwidth`. This dependency would correctly return the unicode length of any special character.

**Comment by ayonious** [comment]
fixed in version 2.7.2

**Comment by ayonious** [comment]
reopened This issue as the package used has some unnecessary code that is making the whole size more than twice

**Comment by ayonious** [comment]
fix released with version 2.7.5
